manual/manual/macros.hva

```diff
@@ -6,6 +6,16 @@
 \newstyle{a:visited}{color:\visited@color;text-decoration:underline;}
 \newstyle{a:hover}{color:black;text-decoration:none;background-color:\hover@color}
 
+% Compact layout
+\newstyle{body}{max-width:800px}
+\newstyle{@media (min-width:900px)}{body\{margin-left:50px\}}
+\newstyle{@media (min-width:1000px)}{body\{margin-left:100px\}}
```

Comment:
Could we have auto margins for left and right rather than 100px? It would center the body, which is more comfortable IMO.

```
\newstyle{@media (min-width:900px)}{body\{margin-left:auto;margin-right: auto;\}}
```

I think it would allow to remove the different conditions for the 900px and 100px min-width too.

Reply:
I'm hoping that in the future the manual will look more like [Racket's](https://docs.racket-lang.org/guide/syntax-overview.html), which I think looks better if the body is left-aligned, so this was partly in hope of making it visually consistent with (possible) future changes. I also think it looks better by itself. I don't particularly mind changing it if others like it better centered though.

Reply:
I uploaded a [centered version](http://sgt.hootr.club/ocaml-manual-pv/), and here's the [left-aligned one](https://nojb.github.io/ocaml-manual/new).